taprpc/marshal.go

```diff
@@ -585,9 +585,11 @@ func MarshalAcceptedBuyQuoteEvent(
 		Id:          event.ID[:],
 		Scid:        uint64(event.ShortChannelId()),
 		AssetAmount: event.Request.AssetAmount,
-		// TODO(ffranr): Temp solution.
-		AskPrice: event.AssetRate.Coefficient.ToUint64(),
-		Expiry:   event.Expiry,
+		AskAssetRate: &rfqrpc.FixedPoint{
```

Comment:
Same here, the constructors should always be used. This is how we make sure that `Coefficient` is the right size internally. We want to scale up with the constructor, not before.

Reply:
Would you like me to create a marshal constructor for the fixed-point RPC type?

Reply:
Yeah, maybe just a `rfqrpc.MarshalFixedPoint` that does exactly this. Just to make sure we don't accidentally forget the scale or coefficient (however unlikely that is).